src/agent/src/main.c

```diff
@@ -831,7 +831,7 @@ void ShutdownAgent()
  */
 void OnShutdownSignal(int sig)
 {
-    Log_Warn("Shutdown signal detected: %d", sig);
+    Log_Warn("Shutdown signal detected: %s", sig);
```

Comment:
the fix here seems to be regressing a previous fix. It should be %d to avoid undefined behavior (including segfault)

Reply:
I've addressed this in https://github.com/Azure/iot-hub-device-update/commit/34e6d5cd2f25934b9370fc351c9532e663b741ca